Message/RegisterViewController.swift

```diff
@@ -42,33 +48,46 @@ class RegisterViewController: UIViewController {
         
         switch resultEmail {
         case .valid:
-            userRepository.register(user: username, password: password, fullname: fullname, completion: nil)
+            userRepository.register(user: username, password: password, fullname: fullname, completion: { result, err in
```

Comment:
```suggestion
            userRepository.register(user: username, password: password, fullname: fullname) { result, err in
```
Dùng trailing closure